aerospike/aerospike_test.go

```diff
@@ -0,0 +1,184 @@
+package aerospike
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/aerospike/aerospike-client-go/v8"
+	"github.com/stretchr/testify/require"
+	"github.com/testcontainers/testcontainers-go"
+	"github.com/testcontainers/testcontainers-go/wait"
+)
+
+const (
+	// aerospikeImage is the default image used for running Aerospike in tests.
+	aerospikeImage              = "aerospike/aerospike-server:latest"
+	aerospikeImageEnvVar string = "TEST_AEROSPIKE_IMAGE"
+	aerospikePort               = "3000/tcp"
+	febricPort                  = "3001/tcp"
+	heartbeatPort               = "3002/tcp"
+	infoPort                    = "3003/tcp"
+	aerospikeReadyLog           = "migrations: complete"
+	aerospikeNamespace          = "test"
+)
+
+// startAerospikeContainer starts an Aerospike container for testing
+func startAerospikeContainer(ctx context.Context) (testcontainers.Container, error) {
+	// Get custom image from env if specified
+	image := aerospikeImage
+	if envImage := os.Getenv(aerospikeImageEnvVar); envImage != "" {
+		image = envImage
+	}
+
+	// Container config
+	req := testcontainers.ContainerRequest{
+		Image:        image,
+		ExposedPorts: []string{aerospikePort, febricPort, heartbeatPort, infoPort},
+		WaitingFor: wait.ForAll(
+			wait.ForLog(aerospikeReadyLog),
+			wait.ForListeningPort(aerospikePort).WithStartupTimeout(5*time.Second),
+			wait.ForListeningPort(febricPort).WithStartupTimeout(5*time.Second),
+			wait.ForListeningPort(heartbeatPort).WithStartupTimeout(5*time.Second),
+		),
+		Cmd: []string{
+			"--config-file",
+			"/etc/aerospike/aerospike.conf",
+		},
+	}
+
+	// Start container
+	container, err := testcontainers.GenericContainer(ctx, testcontainers.GenericContainerRequest{
+		ContainerRequest: req,
+		Started:          true,
+	})
+	if err != nil {
+		return nil, fmt.Errorf("failed to start aerospike container: %w", err)
+	}
+
+	return container, nil
+}
+
+// setupTestClient creates a client connected to the test container
+func setupTestClient(t *testing.T) *Storage {
```

Comment:
suggestion: let's use the same function name that is used across the different modules, for consistency. Please apply it to all references
```suggestion
// newTestStore creates a client connected to the test container
func newTestStore(t *testing.T) *Storage {
```